From f88c73cf278da9907496eab0777903f942c50cb9 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Tue, 21 Apr 2020 10:34:49 +0200
Subject: [PATCH] Fix build with Qt 5.15 (missing header)

Test Plan: Build succeeds with Qt 5.15.0_beta4

Reviewers: #konversation, psn

Reviewed By: #konversation, psn

Differential Revision: https://phabricator.kde.org/D29038
---
 src/viewer/viewtree.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/viewer/viewtree.cpp b/src/viewer/viewtree.cpp
index eeee8fb4..04ec4e54 100644
--- a/src/viewer/viewtree.cpp
+++ b/src/viewer/viewtree.cpp
@@ -21,6 +21,7 @@
 #include <QFontDatabase>
 #include <QGuiApplication>
 #include <QPainter>
+#include <QPainterPath>
 #include <QItemSelectionModel>
 #include <QStyleHints>
 #include <QToolTip>
-- 
2.26.1

